app/assets/stylesheets/form.scss

```diff
@@ -1,3 +1,9 @@
+$green: #94B24D;
+$light_green: #D3FF82;
+$blue: #363D52;
+$dark_blue: #121D2B;
```

Comment:
Color `#121D2B` should be written as `#121d2b`<br>Name of variable `dark_blue` should be written in all lowercase letters with hyphens instead of underscores